### PR TITLE
Ngap

### PR DESCRIPTION
**Pull request by ndp-opendap**
NGAP PI-20

### CODE REVIEW

**Comment by jgallagher59701** [comment]
Merge

File: modules/dmrpp_module/CredentialsManager.cc [hunk 1]
```diff
@@ -25,6 +25,7 @@
 
 #include "config.h"
 
+#include <stdlib.h>
```
Comment (jgallagher59701) [inline modules/dmrpp_module/CredentialsManager.cc]: #include <cstdlib>

Comment (jgallagher59701) [inline modules/dmrpp_module/CredentialsManager.cc]: `Use #include <cstdlib>`

Comment (ndp-opendap) [reply modules/dmrpp_module/CredentialsManager.cc]: done

**Pull request merged**